### DISCUSSION TITLE
How can I get jira status list?

### DISCUSSION THREAD

**Discussion by Wangggym** [ankitpokhrel/jira-cli#677] [Ideas] [answered]
> Please consider [opening a discussion](https://github.com/ankitpokhrel/jira-cli/discussions/categories/ideas) in `ideas` category if applicable. Your feature will be prioritized on based on the number of votes.  

**Is your feature request related to a problem? Please describe.**

In my project, I use github-cli and jira-cli to complete the quick workflow. See https://github.com/Wangggym/quick-workflow
and I want to get the Jira status list in different project.

**Describe the solution you'd like**

like is better:
```shell
jira project status list xxx
```
and return:
```shell
['In progress','Done']
```

**Describe alternatives you've considered**

**Additional context**

**Comment by Wangggym** [comment]
Why didn't there be a reply to the message

**Reply by ankitpokhrel** [reply]
Hi @Wangggym This is not possible at the moment and out of scope of the project. You would need to make a curl request for json response.

**Reply by Wangggym** [reply]
But I know the Abbreviation of the project and in your CLI tools  I use `jira issue move`  if is incorrect status will return the status list.
This is very important for my project and thank you for your answer. @ankitpokhrel

**Reply by Wangggym** [reply]
By the way, how is the curl data structure? thanks

**Comment by Wangggym** [comment, marked as answer]
https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-projects/#api-rest-api-3-project-projectidorkey-statuses-get
I found this API and it will help me complete my project, but I still hope to use it in your CLI tools. I even want to contribute code, but unfortunately I don't know the `go` language.
